### PR TITLE
Updated few of the sentences

### LINKED ISSUE #20306: Can't read this
[Enter feedback here]

There's quite a lot here that's not real English.

"Just for the example sake, you can see one of the above policies in the next code:"

and then it's followed by something that's quite irrelevant to what's being discussed, with some mentions of Polly which is also irrelevant in the context.

This has to be the longest sentence ever written:
"Up to this point, the code shown is just performing regular Http requests, but the 'magic' comes in the following sections where, just by adding policies and delegating handlers to your registered Typed Clients, all the HTTP requests to be done by HttpClient will behave taking into account resilient policies such as retries with exponential backoff, circuit breakers, or any other custom delegating handler to implement additional security features, like using auth tokens, or any other custom feature."

Humour me and actually read through it, now tell me what it's trying to say. I stopped reading after the first five hundred or so characters as I can't wrap my head around it.

The *only* thing I wanted to see is how to consume strongly typed clients - ie how to inject them into your named service (is it implicitly done when using the client factory in the appropriate service?). Either it's not here or it's lost in all the irrelevant and overly long sections.

Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7b506236-b211-eb9d-b571-5efecdde76c8
* Version Independent ID: 24434b0e-0b82-e477-9440-2cc8f135df75
* Content: [Use IHttpClientFactory to implement resilient HTTP requests](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests)
* Content Source: [docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md](https://github.com/dotnet/docs/blob/master/docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md)
* Product: **dotnet-architecture**
* Technology: **microservices**
* GitHub Login: @nishanil
* Microsoft Alias: **nanil**

### PR DESCRIPTION
**Pull request by sughosneo**
## Summary

Fixes #20306 - Simplify the sentences and minor changes.

### CODE REVIEW

File: docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md [hunk 1]
```diff
@@ -145,13 +145,13 @@ public class CatalogService : ICatalogService
 }
 ```
 
-The Typed Client (`CatalogService` in the example) is activated by DI (Dependency Injection), meaning that it can accept any registered service in its constructor, in addition to `HttpClient`.
+The Typed Client (`CatalogService` in the example) is activated by DI (Dependency Injection), that means it can accept any registered service in its constructor, in addition to `HttpClient`.
 
-A Typed Client is, effectively, a transient object, meaning that a new instance is created each time one is needed and it will receive a new `HttpClient` instance each time it's constructed. However, the `HttpMessageHandler` objects in the pool are the objects that are reused by multiple `HttpClient` instances.
+A Typed Client is effectively a transient object, that means a new instance is created each time one is needed. And it will receive a new `HttpClient` instance each time it's constructed. However, the `HttpMessageHandler` objects in the pool are the objects that are reused by multiple `HttpClient` instances.
```
Comment (IEvangelist) [inline docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md:150]:
```suggestion
A Typed Client is effectively a transient object, that means a new instance is created each time one is needed. It receives a new `HttpClient` instance each time it's constructed. However, the `HttpMessageHandler` objects in the pool are the objects that are reused by multiple `HttpClient` instances.
```

No need to start the sentence with "And". Also, use [active voice](https://docs.microsoft.com/en-us/contribute/dotnet/dotnet-voice-tone#use-active-voice).

File: docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md [hunk 2]
```diff
@@ -180,7 +180,7 @@ namespace Microsoft.eShopOnContainers.WebMVC.Controllers
 }
 ```
 
-Up to this point, the code shown is just performing regular Http requests, but the 'magic' comes in the following sections where, just by adding policies and delegating handlers to your registered Typed Clients, all the HTTP requests to be done by `HttpClient` will behave taking into account resilient policies such as retries with exponential backoff, circuit breakers, or any other custom delegating handler to implement additional security features, like using auth tokens, or any other custom feature.
+Up to this point, the above code snippet has only shown the example of performing regular Http requests. But the 'magic' comes in the following sections where it shows how all the HTTP requests made by `HttpClient`, can have resilient policies such as retries with exponential backoff, circuit breakers, security features using auth tokens, or even any other custom feature. And all of these can be done just by adding policies and delegating handlers to your registered Typed Clients.
```
Comment (IEvangelist) [inline docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md:183]:
```suggestion
Up to this point, the above code snippet has only shown the example of performing regular HTTP requests. But the 'magic' comes in the following sections where it shows how all the HTTP requests made by `HttpClient`, can have resilient policies such as retries with exponential backoff, circuit breakers, security features using auth tokens, or even any other custom feature. And all of these can be done just by adding policies and delegating handlers to your registered Typed Clients.
```

Correct casing.

**Review by IEvangelist — CHANGES_REQUESTED** [review]
Hi @sughosneo - these are awesome improvements, well done. I have a few minor corrections.

File: docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md (hunk 1 above)
Comment (sughosneo) [reply docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md:150]: Thank you @IEvangelist for your review and comments. Will incorporate these suggestions.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

**Comment by sughosneo** [comment]
> Looks good to me, :shipit: when you're ready.

Thanks @IEvangelist.

**Pull request merged**